docs/migration-guide.rst

```diff
@@ -23,12 +23,14 @@ Rasa 1.2 to Rasa 1.3
 
 General
 ~~~~~~~
-
+- Default parameters and architectures for both ``EmbeddingPolicy`` and
```

Comment:
This also needs information about the configuration values. can some of them be reused? should also add a pointer to the docs of the config values